src/OrchardCore.Modules/OrchardCore.Search.Azure.CognitiveSearch/Controllers/AdminController.cs

```diff
@@ -0,0 +1,379 @@
+    {
+        if (!await _authorizationService.AuthorizeAsync(User, AzureCognitiveSearchIndexPermissionHelper.ManageAzureCognitiveSearchIndexes))
+        {
+            return Forbid();
+        }
+
+        var indexes = (await _cognitiveSearchIndexSettingsService.GetSettingsAsync()).Select(i => new IndexViewModel { Name = i.IndexName }).ToList();
+
+        var totalIndexes = indexes.Count;
+
+        if (!string.IsNullOrWhiteSpace(options.Search))
+        {
+            indexes = indexes.Where(q => q.Name.Contains(options.Search, StringComparison.OrdinalIgnoreCase)).ToList();
+        }
+
+        var siteSettings = await _siteService.GetSiteSettingsAsync();
+        var pager = new Pager(pagerParameters, siteSettings.PageSize);
+
+        indexes = indexes
+            .Skip(pager.GetStartIndex())
+            .Take(pager.PageSize).ToList();
+
+        // Maintain previous route data when generating page links
+        var routeData = new RouteData();
+        var pagerShape = await _shapeFactory.CreateAsync("Pager", Arguments.From(new
+        {
+            pager.Page,
+            pager.PageSize,
+            TotalItemCount = totalIndexes
+        }));
+
+        var model = new AdminIndexViewModel
+        {
+            Indexes = indexes,
+            Options = options,
+            Pager = pagerShape
+        };
+
+        model.Options.ContentsBulkAction = [
+            new SelectListItem(S["Delete"], nameof(ContentsBulkAction.Remove)),
+        ];
+
+        return View(model);
+    }
+
+    [HttpPost, ActionName(nameof(Index))]
+    [FormValueRequired("submit.Filter")]
+    public ActionResult IndexFilterPOST(AdminIndexViewModel model)
+    {
+        return RedirectToAction("Index", new RouteValueDictionary {
+                { "Options.Search", model.Options.Search }
+            });
+    }
+
+    [HttpPost, ActionName(nameof(Index))]
+    [FormValueRequired("submit.BulkAction")]
+    public async Task<ActionResult> IndexPost(ContentOptions options, IEnumerable<string> itemIds)
+    {
+        if (!await _authorizationService.AuthorizeAsync(User, AzureCognitiveSearchIndexPermissionHelper.ManageAzureCognitiveSearchIndexes))
+        {
+            return Forbid();
+        }
+
+        if (itemIds?.Count() > 0)
+        {
+            var indexSettings = await _cognitiveSearchIndexSettingsService.GetSettingsAsync();
+            var checkedContentItems = indexSettings.Where(x => itemIds.Contains(x.IndexName));
+
+            switch (options.BulkAction)
+            {
+                case ContentsBulkAction.None:
+                    break;
+                case ContentsBulkAction.Remove:
+                    foreach (var item in checkedContentItems)
+                    {
+                        await _azureCognitiveSearchIndexManager.DeleteAsync(item.IndexName);
+                    }
+                    await _notifier.SuccessAsync(H["Indices successfully removed."]);
```

Comment:
```suggestion

                    await _notifier.SuccessAsync(H["Indices successfully removed."]);
```

Reply:
Add a line before

Reply:
Why do you need and extra line before? Typically we add a new line before return not after every line.

Reply:
Also, after the end of any block. Please check the ASP.NET Core code conventions which we already used since the beginning

Reply:
@hishamco I think you are not looking at the latest code. there was a new line there already. Maybe go back and refresh the page.